devtools/gradle/src/main/java/io/quarkus/gradle/builder/QuarkusModelBuilder.java

```diff
@@ -56,13 +56,21 @@
     private static final String MAIN_RESOURCES_OUTPUT = "build/resources/main";
     private static final String CLASSES_OUTPUT = "build/classes";
     private static final String DEPLOYMENT_CONFIGURATION = "deploymentConfiguration";
+    private static final String CLASSPATH_CONFIGURATION = "classpathConfiguration";
```

Comment:
Is this a Quarkus-specific classpath config?

Reply:
Yes this the configuration I to combine `quarkusDev` / `compileOnly` / `runtime`dependencies

Reply:
Perhaps we should call it `quarkus`SomethingConfiguration then.